tutorials/advanced_features/Virtual Devices.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>Maybe add a subtitle so that the <code>replace()</code> feature is highlighted</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>